docker-set-symlinks.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright 2018-present Samsung Electronics Co., Ltd. and other contributors
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ -e /dev/ttyACM0 ]
+then
+    ln -s /dev/ttyACM0 /dev/STM32F4
+fi
+
+if [ -e /dev/ttyUSB0 ]
+then
+    ARTIK_FILE=/dev/ttyUSB0
+elif [ -e /dev/ttyUSB1 ]
+then
+    ARTIK_FILE=/dev/ttyUSB0
+fi
+
+ln -s $ARTIK_FILE /dev/ARTIK053
```

Comment:
This line should be moved into the appropriate `if` statements. (If neither `/dev/ttyUSB0` nor `/dev/ttyUSB1` doesn't exist, `$ARTIK_FILE` value is empty.)

Reply:
It brings another question. If you only have one serial device, it will always be ttyUSB0, right?
Why we also want to handle `ttyUSB1`? If it is possible to get another number which is different from zero, then what happens if the order that the device is detected is greater than `1`?

Reply:
It wasn't always `ttyUSB0` on my PC, but it has never been anything else than `ttyUSB0` or `ttyUSB1`. But of course we should handle other cases. If the user only connects one device to the docker container at a time, than we can simply look for any device that starts with `ttyUSB` and set up the symlink for that. So something like this:
```sh
ARTIK_FILE=$(ls /dev | grep "ttyUSB")
ln -s $ARTIK_FILE /dev/ARTIK053
```